store/snap_remote_repo.go

```diff
@@ -148,26 +174,27 @@ func NewUbuntuStoreSnapRepository(cfg *SnapUbuntuStoreConfig, storeID string) *S
 	}
 	// see https://wiki.ubuntu.com/AppStore/Interfaces/ClickPackageIndex
 	return &SnapUbuntuStoreRepository{
-		storeID:    storeID,
-		searchURI:  cfg.SearchURI,
-		detailsURI: cfg.DetailsURI,
-		bulkURI:    cfg.BulkURI,
+		storeID:       storeID,
+		searchURI:     cfg.SearchURI,
+		detailsURI:    cfg.DetailsURI,
+		bulkURI:       cfg.BulkURI,
+		assertionsURI: cfg.AssertionsURI,
 	}
 }
 
-// small helper that sets the correct http headers for the ubuntu store
-func (s *SnapUbuntuStoreRepository) setUbuntuStoreHeaders(req *http.Request) {
-	var token *oauth.Token
-	ssoToken, _ := ReadStoreToken()
-	if ssoToken != nil {
-		token = &ssoToken.Token
+// setAuthHeader sets the authorization header
+func (s *SnapUbuntuStoreRepository) setAuthHeader(req *http.Request, token *StoreToken) {
```

Comment:
Just a passing comment really ... but *to me* setAuthHeader suggests it's setting something on the receiver, i.e. the store repo, but I think it only configures the `req` arg, right? Something like configureReqAuthHeader() might read a little better.